src/DurableTask.SqlServer.AzureFunctions/SqlDurabilityProviderFactory.cs

```diff
@@ -82,14 +82,13 @@ public DurabilityProvider GetDurabilityProvider(DurableClientAttribute attribute
                 }
 
                 SqlDurabilityOptions clientOptions = this.GetSqlOptions(attribute);
-                IOrchestrationServiceClient serviceClient = 
-                    new SqlOrchestrationService(clientOptions.GetOrchestrationServiceSettings(
+                SqlOrchestrationService orchestrationService = 
+                    new (clientOptions.GetOrchestrationServiceSettings(
```

Comment:
Should this instead be `SqlOrchestrationService orchestrationService = this.GetOrchestrationService()`?

Reply:
Nope the problem come from this.GetOrchestrationService() that is not taking care of give options